gini/gini-vision-lib-android
# Finalized QR Code detected popup UI

QR Code detected popup UI updated.

### How to test
Run an example app and move the camera over a BezahlCode or StuzzaCode or GiroCode and check the popup.

### Ticket 
#136
